app/representers/api/podcast_representer.rb

```diff
@@ -28,6 +28,7 @@ class Api::PodcastRepresenter < Api::BaseRepresenter
   property :locked
   property :payment_pointer
   property :donation_url
+  property :guid
```

Comment:
Adding it here as well, for completeness (and if we want to sync via API we could)